link/src/test/java/com/stripe/android/link/ui/paymentmethod/PaymentMethodViewModelTest.kt

```diff
@@ -200,6 +201,59 @@ class PaymentMethodViewModelTest {
             )
         }
 
+    @Test
+    fun `when shippingValues are passed ConfirmStripeIntentParams has shipping`() = runTest {
+        val value = createLinkPaymentDetails()
+        whenever(
+            linkAccountManager.createCardPaymentDetails(anyOrNull(), anyOrNull(), anyOrNull())
+        ).thenReturn(Result.success(value))
+        whenever(navigator.isOnRootScreen()).thenReturn(false)
+        whenever(args.shippingValues).thenReturn(
+            mapOf(
+                IdentifierSpec.Name to "Test Name",
+                IdentifierSpec.Country to "US"
+            )
+        )
+
+        createViewModel().startPayment(cardFormFieldValues)
+
+        val paramsCaptor = argumentCaptor<ConfirmStripeIntentParams>()
+        verify(confirmationManager).confirmStripeIntent(paramsCaptor.capture(), any())
+
+        assertThat(paramsCaptor.firstValue.toParamMap()).isEqualTo(
+            mapOf(
+                "client_secret" to args.stripeIntent.clientSecret,
+                "use_stripe_sdk" to false,
+                "mandate_data" to mapOf(
+                    "customer_acceptance" to mapOf(
+                        "type" to "online",
+                        "online" to mapOf(
+                            "infer_from_client" to true
+                        )
+                    )
+                ),
+                "shipping" to mapOf(
+                    "address" to mapOf(
+                        "country" to "US"
+                    ),
+                    "name" to "Test Name"
+                ),
+                "payment_method_data" to mapOf(
+                    "type" to "link",
+                    "link" to mapOf(
+                        "payment_details_id" to "QAAAKJ6",
+                        "credentials" to mapOf(
+                            "consumer_session_client_secret" to CLIENT_SECRET
+                        ),
+                        "card" to mapOf(
+                            "cvc" to "123"
+                        )
+                    )
+                )
+            )
+        )
```

Comment:
Would it make sense to only assert the part under `"shipping"` here to be more in line with the test name?